src/domains/ecommerce-app/api/ecommerce-webview/v1/_openapi.json.tpl

```diff
@@ -378,6 +378,15 @@
         ],
         "operationId": "createWalletForTransactionsForIO",
         "parameters": [
+          {
+            "name": "webSessionToken",
```

Comment:
```suggestion
            "name": "x-ecommerce-session-token",
```

since we are propagating this information as a custom header I would stick with the x-... naming convention.
moreover I would use a name that recall the fact that this is an eCommerce domain header, just to make it clear the token domain and origin
 what do you think?